packages/react-components/src/components/ProductSwitcher/components/ProductRow/ProductRow.tsx

```diff
@@ -0,0 +1,106 @@
+import { FC, MouseEvent } from 'react';
+import * as React from 'react';
+
+import { Info } from '@livechat/design-system-icons';
+import cx from 'clsx';
+
+import { Icon } from '../../../Icon';
+import { Tooltip } from '../../../Tooltip';
+import { Text } from '../../../Typography';
+import { useProductSwitcherShortcut } from '../../hooks/useProductSwitcherShortcut';
+import { ProductOption, ProductId } from '../../types';
+import { ProductTile } from '../ProductTile/ProductTile';
+
+import styles from './ProductRow.module.scss';
+
+type IProps = {
+  onClick: (event: MouseEvent, id: ProductId) => void;
+  product: ProductOption;
+  isActive?: boolean;
+};
+
+const baseClass = 'product-row';
+
+export const ProductRow: FC<IProps> = ({
+  product: {
+    id,
+    icon,
+    iconColor,
+    backgroundColors,
+    notificationCount,
+    name,
+    url,
+    withDivider,
+    shortcutKey,
+    expired,
+    trialDaysLeft,
+    nameAdornment,
+  },
+  onClick,
+  isActive = false,
+}) => {
+  if (shortcutKey) {
+    useProductSwitcherShortcut(shortcutKey, url);
+  }
+
+  return (
+    <>
+      {withDivider && <div className={styles[`${baseClass}__divider`]} />}
+      <a
+        href={url}
+        target="_blank"
```

Comment:
Shouldn't this be configurable? 🤔

Reply:
AFAIK, there is no such requirement now - switcher should always open the product in a new tab.

Reply:
I guess we want to have a consistent component across all of the products, so it would be better to keep it not configurable